src/js/components/Widgets/SignInModal.jsx

```diff
@@ -207,26 +250,37 @@ const styles = theme => ({
       height: 'unset',
       margin: '0 auto',
     },
-  } : {
-    margin: '0 !important',
-    width: '95%',
-    height: 'unset',
-    maxHeight: '90%',
-    offsetHeight: 'unset !important',
-    top: '50%',
-    left: '50%',
-    right: 'unset !important',
-    bottom: 'unset !important',
-    position: 'absolute',
-    transform: 'translate(-50%, -25%)',
   },
-  focusedOnSingleInput: isWebApp() ? {
-    [theme.breakpoints.down('sm')]: {
-      position: 'absolute',
-      top: '75%',
-      left: '73%',
-    },
-  } : {},
+  // dialogPaper: isWebApp() ? {
+  //   [theme.breakpoints.down('sm')]: {
+  //     minWidth: '95%',
+  //     maxWidth: '95%',
+  //     width: '95%',
+  //     maxHeight: '90%',
+  //     height: 'unset',
+  //     margin: '0 auto',
+  //   },
+  // } : {
+  //   margin: '0 !important',
+  //   width: '95%',
+  //   height: 'unset',
+  //   maxHeight: '90%',
+  //   offsetHeight: 'unset !important',
+  //   top: '50%',
+  //   left: '50%',
+  //   right: 'unset !important',
+  //   bottom: 'unset !important',
+  //   position: 'absolute',
+  //   transform: 'translate(-50%, -58%)',
+  // },
+  // focusedOnSingleInput: isWebApp() ? {
```

Comment:
I believe we still need this first `focusedOnSingleInput` branch for WebApp. What do you think @SailingSteve?

Reply:
same, will back out.